Frontend/src/components/DummyComponent/DummyComponent.module.css

```diff
@@ -1,11 +1,27 @@
-.headingContainer{
```

Comment:
Is this going to be responsive?

Reply:
To an extent but I can make certain tweaks for it's responsiveness.